db/models/risk.js

```diff
@@ -44,13 +52,17 @@ const RisksSchema = new Schema({
         type: Number,
         min: 1,
         max: 25
```

Comment:
Can you change the min to 2 and max to 10

Reply:
changes done as needed